docs/source/samples/UVC/uvc_rgb.rst

```diff
@@ -1,7 +1,7 @@
 UVC & Color Camera
 ==================
 
-This example demonstrates how to use the RGB camera on your OAK device as a UVC webcam. The UVC feature allows you to use your OAK device as a regular webcam in applications like OpenCV's :code:`cv2.VideoCapture()`, native camera apps, and more.
+This example demonstrates how to use the RGB camera on your OAK device as a UVC webcam. The :ref:`UVC` feature allows you to use your OAK device as a regular webcam in applications like OpenCV's :code:`cv2.VideoCapture()`, native camera apps, and more.
```

Comment:
Docs failing to build because:
```
/home/docs/checkouts/readthedocs.org/user_builds/luxonis-depthai-python/checkouts/917/docs/source/samples/UVC/uvc_disparity.rst:8: WARNING: undefined label: uvc
/home/docs/checkouts/readthedocs.org/user_builds/luxonis-depthai-python/checkouts/917/docs/source/samples/UVC/uvc_mono.rst:8: WARNING: undefined label: uvc
/home/docs/checkouts/readthedocs.org/user_builds/luxonis-depthai-python/checkouts/917/docs/source/samples/UVC/uvc_rgb.rst:4: WARNING: undefined label: uvc
/home/docs/checkouts/readthedocs.org/user_builds/luxonis-depthai-python/checkouts/917/docs/source/tutorials/code_samples.rst:76: WARNING: undefined label: feature motion estimation
```